src/Layout.tsx

```diff
@@ -91,13 +91,24 @@ const Layout = () => {
   }, [channelId])
 
   return (
-    <main className="h-screen flex pt-6 md:pt-0 md:items-center justify-center font-sans bg-slate-50">
-      <Outlet context={{ channelName: channelId, clientId, setProjectInfo }} />
-      <div className="fixed bottom-0 md:absolute md:left-12 md:bottom-12">
-        <InfoCard projectInfo={projectInfo} />
-      </div>
-    </main>
-  )
-}
+    <>
+      {projectInfo.topic === 'emoji-reactions' || projectInfo.topic === 'user-claims' || projectInfo.topic === 'avatar-stack' ? (
```

Comment:
should this contain the default `realtime-examples` too?

A different way to write this would to use `includes`:

```js
const oldLayout = ['emoji-reactions', 'user-claims', ...];
...
oldLayout.includes(projectInfo.topic)
```